code/modules/client/asset_cache.dm

```diff
@@ -280,7 +280,7 @@ GLOBAL_LIST_EMPTY(asset_datums)
 	)
 
 /datum/asset/simple/goonchat
-	verify = TRUE
+	verify = FALSE

```

Comment:
Minor thing, but you could just scrap the line entirely since it defaults to false